hummingbot/market/binance/binance_time.py

```diff
@@ -30,60 +28,52 @@ def get_instance(cls) -> "BinanceTime":
         return cls._bt_shared_instance
 
     def __init__(self, check_interval: float = 60.0):
-        self._time_offset_ms = deque([])
+        self._time_offset_ms = 0.0
         self._set_server_time_offset_task = None
         self._started = False
         self.SERVER_TIME_OFFSET_CHECK_INTERVAL = check_interval
-        self.median_window = 100
 
     @property
     def started(self):
         return self._started
 
     @property
     def time_offset_ms(self):
-        if not self._time_offset_ms:
-            return 0.0
-        return statistics.median(self._time_offset_ms)
```

Comment:
The statistical error method is still needed, to tackle inconsistent network latencies.

Instead of removing the stats mechanism to tackle system clock getting set forward or backward... We can use monotonic clocks for local clock reference instead.

I.e. instead of `time.time()`, we can use `time.perf_counter()` instead.